docs/docs/providers/42.md

```diff
@@ -13,7 +13,9 @@ https://api.intra.42.fr/apidoc/guides/web_application_flow
 
 ## Configuration
 
-https://profile.intra.42.fr/oauth/applications/new
+https://profile.intra.42.fr/oauth/applications/new  
+The callback uri has to be  
+DOMAIN.TLD/api/auth/callback/42-school
```

Comment:
We should highlight this better in the **Getting Started** docs instead 🤔 
https://next-auth.js.org/getting-started/rest-api#getpost-apiauthcallbackprovider

cc. @balazsorban44

Reply:
FYI https://github.com/nextauthjs/next-auth/pull/4065

Reply:
Let's remove this part as well; I opened a PR to explain this part better in the **Getting Started** tutorial.